ros/processor/event_producer.py

```diff
@@ -8,12 +8,11 @@
 logger = get_logger(__name__)
 
 
-def new_suggestion_event(host, platform_metadata, previous_state, current_state, producer):
+def notification_payload(host, previous_state, current_state):
```

Comment:
should we use same naming for parameters? as `system_previous_state` and `system_current_state`?

Reply:
Done